kore/src/SMT/SimpleSMT.hs

```diff
@@ -210,28 +210,31 @@ import Prelude.Kore hiding (
     not,
     or,
  )
+import qualified Prelude.Kore as Prelude
```

Comment:
Why is this imported qualified?

Reply:
The SimpleSMT module defines some overlapping function names: `not`, `or`, `div`, etc. See `import Prelude.Kore hiding (...` just above line 213 for a complete list. To disambiguate those functions the import needs to be qualified. This change is just replacing `import qualified Prelude as Prelude` with `import qualified Prelude.Kore as Prelude`. Either editorconfig or fourmolu rearranged the imports so the change isn't obvious on Github.

Reply:
Alright, sounds good then!

Reply:
My reasoning for the change from `Prelude` to `Kore.Prelude` is for consistency; we should always prefer to use our custom prelude over the standard prelude.

Reply:
Yeah, definitely.